### PR TITLE
1659 Use color variables

### PR DESCRIPTION
**Pull request by ericsorenson**

### CODE REVIEW

**Review by codyseibert — APPROVED** [review]

File: web-client/src/styles/custom.scss [hunk 1]
```diff
@@ -863,8 +863,8 @@ nav.usa-nav {
 .big-blue-header {
   padding: 25px 0;
   margin-bottom: 40px;
-  background-color: #162e51; // TODO: Replace with USWDSv2 var when available;
-  color: $color-white; // TODO: Replace with USWDSv2 var when available;
+  background-color: color('primary-darker');
```
Comment (rachaelparris) [inline web-client/src/styles/custom.scss]: Should be `color($theme-color-primary-darker)`

**Review by rachaelparris — CHANGES_REQUESTED** [review]

File: web-client/src/styles/custom.scss (hunk 1 above)
Comment (rachaelparris) [reply web-client/src/styles/custom.scss]: I tried this locally and it's working like this, so did I do the color updates incorrectly? I can go back and change them to be like this if we'd prefer it this way.

Comment (ericsorenson) [reply web-client/src/styles/custom.scss]: thanks

**Review by rachaelparris — APPROVED** [review]

**Pull request merged**